x-pack/plugins/case/server/routes/api/cases/post_case.ts

```diff
@@ -34,6 +39,11 @@ export function initPostCaseApi({ caseService, router, userActionService }: Rout
 
         const { username, full_name, email } = await caseService.getUser({ request, response });
         const createdDate = new Date().toISOString();
+        const myCaseConfigure = await caseConfigureService.find({ client });
+        const connectorId =
```

Comment:
it might be useful to create a function to get the connector_id, so if we had to change it will be just in one place.